assets/components/legal/subscriptionTermsPrivacy/subscriptionTermsPrivacy.scss

```diff
@@ -0,0 +1,13 @@
+.component-subscription-terms-privacy__text {
+  color: gu-colour(neutral-4);
+  display: block;
+}
+
+.component-subscription-terms-privacy__href {
```

Comment:
Worth making this more generic so that all links on the page get the same treatment? It'll also mean we can remove `subscriptionFaq.scss`

Reply:
I thought about putting the styles for links into `footer.scss` instead and then adding `className=component-footer__href` to the links but I wasn't sure if that's where those styles should live. Is that what you're suggesting, or something else?

Reply:
Either way, I think I'm going to merge this tomorrow morning, but possibly followed by a little PR to genericise after.